app/views/partials/_notification.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if flash[:notice] %>
+  <div id="notice_wrapper" class="notification is-success has-text-centered">
+    <button class="delete"></button>
+    <%= notice %>
+  </div>
+<% end %>
+<% if flash[:alert] %>
+  <div id="notice_wrapper" class="notification is-danger has-text-centered">
+    <button class="delete"></button>
+    <%= alert %>
+  </div>
+<% end %>
+<% if flash[:error] %>
+  <div id="notice_wrapper" class="notification is-danger has-text-centered">
+    <button class="delete"></button>
+    <%= flash[:error] %>
+  </div>
+<% end %>
```

Comment:
Add a line blank

Reply:
because?

Reply:
This is what the Ruby Styles Guide indicates, and using console writing, line blank helps a lot.